packages/pages/src/scaffold/modules/templates.ts

```diff
@@ -0,0 +1,85 @@
+export const moduleCode = (
+  moduleName: string,
+  useTailwind: boolean
+): string => {
+  const tailwind = useTailwind ? ` className="tailwind"` : ``;
+  const formattedModuleName =
+    moduleName.charAt(0).toUpperCase() + moduleName.slice(1);
+
+  return `import { Module, ModuleConfig, ModuleProps } from "@yext/pages/*";
+import { AnalyticsProvider } from "@yext/pages-components";
+import "./index.css";
+
+const templateData: ModuleProps = {
+  document: {
+    businessId: "REPLACE_ME",
+    siteId: "REPLACE_ME",
+    apiKey: "REPLACE_ME",
```

Comment:
Remove, this isn't part of document. The apiKey I was referring to is a top-level prop of AnalyticsProvider once my update is out.

Here are the updated props:
```
constructor(
    private apiKey: string,
    private defaultCurrency: string,
    private templateData: TemplateProps,
    requireOptIn?: boolean | undefined,
    private productionDomains: string[] = [],
    disableSessionTracking?: boolean | undefined,
    private enableDebugging: boolean = false
  )
```

We should collect (aka set "REPLACE_ME"s):
* apiKey
* defaultCurrency
* templateData (with this data of course)
* productionDomains

Reply:
Ah gotcha. Should this item not be released until after ur analytics then? Since apiKey doesn't exist rn?

Reply:
You can just add them. It fails TS validation but that's only in your editor. It still builds fine. Try it out.

Reply:
Yup tested + confirms works fine, just fails IDE TS validation.